src/fmu/dataio/aggregation.py

```diff
@@ -296,13 +305,20 @@ def generate_metadata(
                 a temporary export of the data, and may be time consuming for large
                 data.
 
-            skip_null: If True (default), None values in putput will be skipped
+            skip_null: This input parameter has been deprecated. If set to False,
+              a deprecation warning will be raised.
             **kwargs: See AggregatedData() arguments; initial will be overridden by
```

Comment:
We would eventually like to get away from this pattern too, where a class instance has its state updated so it can be reused. `**kwargs` is present in a few other places as well.